govcd/vapp.go

```diff
@@ -80,7 +92,7 @@ func (vapp *VApp) Refresh() error {
 	return nil
 }
 
-func (vapp *VApp) AddVM(orgvdcnetworks []*types.OrgVDCNetwork, vapptemplate VAppTemplate, name string, acceptalleulas bool) (Task, error) {
+func (vapp *VApp) AddVM(orgVdcNetworks []*types.OrgVDCNetwork, vappNetworkName string, vapptemplate VAppTemplate, name string, acceptalleulas bool) (Task, error) {
```

Comment:
This change allows to add VM with vappNetworkName.  Current implementation will add org vdc networks if they provided. So now vapp network and vdc network will be added if provided.
I will commit one missing check in this function.

Reply:
This function is from old times and I have various questions there and yet it wasn't scope of this issue to analyse and think what needs to be improved at the moment. Currectly it was enhance to support adding vappNetworkName, cause previously it would add only orgVdcNetwork. OrgVcdNetwork is array of structs and vappNetworkName is string. So pointer is just memory efficiency question. I will add comment according I understand regarding this old function.